app/scripts/search-browse/cedar-search-browse-picker.directive.js

```diff
@@ -1405,19 +1405,22 @@ define([
           }
 
           function openOpen(resource) {
+            console.log("Resource: ", resource);
```

Comment:
Are these comments needed in the final code?

Reply:
Removed those line